pkg/kubelet/nvidiagpu/nvidia_gpu_manager.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nvidiagpu
+
+import (
+	"fmt"
+	"sync"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"k8s.io/kubernetes/pkg/kubelet/dockertools"
+	dockertypes "github.com/docker/engine-api/types"
+)
+
+
+// TODO: If use NVML in the future, the implementation could be more complex,
+// but also more powerful!
+
+const (
+	// All NVIDIA GPUs cards should be mounted with nvidiactl and nvidia-uvm
+	NvidiaDeviceCtl string = "/dev/nvidiactl"
+	NvidiaDeviceUVM string = "/dev/nvidia-uvm"
+)
+
+type NvidiaGPUManager struct {
+	gpuPaths []string
+	gpuMutex sync.Mutex
+
+	dockerClient dockertools.DockerInterface
```

Comment:
@feiskyer I think so far GPU as an alpha resource it is ok we do not add a method to the runtime interface.